Dockerfile

```diff
@@ -215,14 +215,19 @@ ARG UAVS3D_COMMIT=073fe7cc8eaef021aa4adc7b0100461e0260b70d
 ARG RUBBERBAND_VERSION=2.0.0
 ARG RUBBERBAND_URL="https://breakfastquay.com/files/releases/rubberband-$RUBBERBAND_VERSION.tar.bz2"
 ARG RUBBERBAND_SHA256=eccbf0545496ce3386a2433ceec31e6576a76ed6884310e4b465003bfe260286
+# bump: libgme /LIBGME_COMMIT=([[:xdigit:]]+)/ gitrefs:https://bitbucket.org/mpyne/game-music-emu.git|re:#^refs/heads/master$#|@commit
+# bump: libgme after ./hashupdate Dockerfile LIBGME $LATEST
+# bump: libgme link "Source diff $CURRENT..$LATEST" https://bitbucket.org/mpyne/game-music-emu/branches/compare/$CURRENT..$LATEST
+ARG LIBGME_URL="https://bitbucket.org/mpyne/game-music-emu.git"
+ARG LIBGME_COMMIT=b3d158a30492181fd7c38ef795c8d4dcfd77eaa9
 
 # -O3 makes sure we compile with optimization. setting CFLAGS/CXXFLAGS seems to override
 # default automake cflags.
 # -static-libgcc is needed to make gcc not include gcc_s as "as-needed" shared library which
 # cmake will include as a implicit library.
 # other options to get hardened build (same as ffmpeg hardened)
-ARG CFLAGS="-O3 -static-libgcc -fno-strict-overflow -fstack-protector-all -fPIE"
-ARG CXXFLAGS="-O3 -static-libgcc -fno-strict-overflow -fstack-protector-all -fPIE"
+ARG CFLAGS="-s -O3 -static-libgcc -fno-strict-overflow -fstack-protector-all -fPIE"
+ARG CXXFLAGS="-s -O3 -static-libgcc -fno-strict-overflow -fstack-protector-all -fPIE"
```

Comment:
I think ffmpeg's build system produces stripped and non-stripped of its tools (the *_g suffixed binaries are not stripped) but im not sure if building the libraries with -s might improve things even more?